README.md

```diff
@@ -0,0 +1,63 @@
+# Craft Platforms
+
+[![QA](https://github.com/canonical/craft-platforms/actions/workflows/qa.yaml/badge.svg)](https://github.com/canonical/craft-platforms/actions/workflows/qa.yaml)
+[![Security scan](https://github.com/canonical/craft-platforms/actions/workflows/security-scan.yaml/badge.svg)](https://github.com/canonical/craft-platforms/actions/workflows/security-scan.yaml)
+[![Documentation](https://github.com/canonical/craft-platforms/actions/workflows/docs.yaml/badge.svg)](https://github.com/canonical/craft-platforms/actions/workflows/docs.yaml)
+[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
+[![PyPI - Version](https://img.shields.io/pypi/v/craft-platforms)](https://pypi.org/project/craft-platforms)
+[![ReadTheDocs badge](https://readthedocs.com/projects/canonical-craft-platforms/badge/?version=latest)](https://canonical-craft-platforms.readthedocs-hosted.com)
```

Comment:
Is there any preference for order? In Snapcraft it's build, docs, Spread, Ruff.

Reply:
I made them alphabetical :-)